.github/workflows/ci.yml

```
@@ -106,6 +105,9 @@ jobs:
         npm run build
       working-directory: ./aiida-registry-app
 
+    - name: Add plugins file to the build folder
+      run: cp aiida-registry-app/src/plugins_metadata.json ./aiida-registry-app/dist/
```

I think I understand why previously the file disappeared. It was wiped by the npm build in the `dist` folder. 
Therefore, I think it is okay here to make the following change:

```suggestion
      run: cp plugins_metadata.json ./aiida-registry-app/dist/
```

---

After this change, the "Move JSON file to the React project" step in `generate_metadata` action can be removed. Can you give it a try?

---

Done, but removing the move step causes an error as the file cannot be accessed.
https://github.com/aiidateam/aiida-registry/actions/runs/6310303316/job/17131993630

---

I see, then everything makes sense. I remember you said before that you are not very sure why need to put the file in both `src` and `dist`. Now we know they are all necessary for building and for attaching to dist on gh_page branch. 
Thus, I think your initial implementation of this PR is 100% correct. 

let's wait preview test to pass and we can merge it.